app/controllers/provider/admin/account/email_configurations_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Provider::Admin::Account::EmailConfigurationsController < Provider::Admin::Account::BaseController
+
+  # We only enable it for master for now, feel free to roll it out for any provider
+  before_action :ensure_master_domain, :email_configurations_enabled?
+
+  def index
+  end
+
+  def new
+    @email_configuration = EmailConfiguration.new
+  end
+
+  def edit
+    @email_configuration = account.email_configurations.find(params[:id])
```

Comment:
if we are master, we probably need to search all configurations, not only owned by provider.

Reply:
pagination

Reply:
I think you mean `pagination` for the index. We will probably add a presenter for this.